examples/model_selection/plot_precision_recall.py

```diff
@@ -5,13 +5,18 @@
 
 Example of Precision-Recall metric to evaluate classifier output quality.
 
-In information retrieval, precision is a measure of result relevancy, while
-recall is a measure of how many truly relevant results are returned. A high
-area under the curve represents both high recall and high precision, where high
-precision relates to a low false positive rate, and high recall relates to a
-low false negative rate. High scores for both show that the classifier is
-returning accurate results (high precision), as well as returning a majority of
-all positive results (high recall).
+Precision-Recall is a useful measure of success of prediction when the
```

Comment:
yeah, and I would change the example title too: "Using precision and recall for classifier evaluation" or something. It's very awkward now.

And maybe replace below "very imbalanced" by "imbalanced"?